src/lib/constants/Messages.ts

```diff
@@ -22,4 +23,90 @@ export const Messages = {
     `• If you installed using our new method there is in alpha-version, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(
       `vtex autoupdate`
     )}.\n`,
+  UPDATE_TOOLBELT_NPM: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `yarn`
+    )}, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(`yarn global add vtex`)}.`,
+  UPDATE_TOOLBELT_BREW: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `brew`
+    )}, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(`brew upgrade vtex/vtex`)}.`,
+  UPDATE_TOOLBELT_STANDALONE: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `AWS Standalone`
+    )}, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(`vtex autoupdate`)}.`,
+  UPDATE_TOOLBELT_CHOCOLATEY: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `chocolatey`
+    )}, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco upgrade vtex`)}.`,
+  DEPRECATE_TOOLBELT_BREW: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `brew`
+    )}, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(`brew unlink vtex && brew install vtex/vtex`)}.`,
+  DEPRECATE_TOOLBELT_STANDALONE: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(`AWS Standalone`)}, update running:
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `curl https://vtex-toolbelt-test.s3.us-east-2.amazonaws.com/uninstall.sh | sh`
+    )}
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+      `curl https://vtex-toolbelt-test.s3.us-east-2.amazonaws.com/install.sh | sh`
+    )}`,
+  DEPRECATE_TOOLBELT_CHOCOLATEY: () =>
+    `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(`chocolatey`)}, update running:
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco uninstall vtex`)}.
+    ${ColorifyConstants.COMMAND_OR_VTEX_REF(`choco install vtex`)}.`,
+}
```

Comment:
I suggest changing from `DEPRECATE_` to `UPDATE_FROM_DEPRECATED` or something that makes more explicit that it's a update instruction, not a deprecate instruction